lib/mpl_toolkits/basemap/__init__.py

```diff
@@ -3206,6 +3206,17 @@ def set_axes_limits(self,ax=None):
             import matplotlib.pyplot as plt
             plt.draw_if_interactive()
 
+
+    def _save_use_hold(self, ax, kwargs):
+        h = kwargs.pop('hold', None)
+        if hasattr(ax, '_hold'):
+            self._tmp_hold = ax._hold
+            ax._hold = h
```

Comment:
My thinking here is that presently (with 2.x), the _hold attribute is always present.  It is set in the AxesBase initializer.  Some time in the future it won't be, and at that point Basemap doesn't need to obey it, and shouldn't set it at all.